app/assets/stylesheets/_buttons.scss

```diff
@@ -69,3 +69,10 @@ a .button,
 
 .orange_submit_button { padding: 5px; }
 .translation_button { @extend .play_video_button; }
+
+.video-js.vjs-big-play-button {
```

Comment:
🐘 but it might be worth adding a comment above these about _why_ we've chosen to hide the default play button.